src/coreComponents/physicsSolvers/SolverBase.cpp

```diff
@@ -812,7 +821,7 @@ real64 SolverBase::nonlinearImplicitStep( real64 const & time_n,
     {
       // cut timestep, go back to beginning of step and restart the Newton loop
       stepDt *= dtCutFactor;
-      GEOS_LOG_LEVEL_RANK_0 ( 1, GEOS_FMT( "New dt = {}", stepDt ) );
+      GEOS_LOG_LEVEL_INFO_RANK_0 ( logInfo::SolverTimeStep, GEOS_FMT( "New dt = {}", stepDt ) );
```

Comment:
The log level doesn't match the old one, why?

Reply:
fixed